components/o-teaser/stories/teaser.stories.tsx

```diff
@@ -0,0 +1,259 @@
+import parse from 'html-react-parser';
+import DOMPurify from 'dompurify';
+
+import withHtml from 'origami-storybook-addon-html';
+import {withDesign} from 'storybook-addon-designs';
+import {ComponentStory, ComponentMeta} from '@storybook/react';
+import {Teaser} from '../src/tsx/teaser';
+import './teaser.scss';
+import {
+	articleArgs,
+	contentPackageArgs,
+	contentPackageItemArgs,
+	opinionArgs,
+	podcastArgs,
+	promotedArgs,
+	topStoryArgs,
+	videoArgs,
+} from './args';
+
+import {argTypes} from './arg-types';
+
+argTypes.customSlot = {
+	type: 'string',
+};
+
+export default {
+	title: 'Components/o-teaser',
+	component: Teaser,
+	decorators: [withDesign, withHtml],
+	parameters: {},
+} as ComponentMeta<typeof Teaser>;
+
+const Story: ComponentStory<typeof Teaser> = args => {
+	console.log();
+
+	if (args.showCustomSlot) {
```

Comment:
```suggestion
	if (typeof args.showCustomSlot === 'string') {
```

It worked!? Nice! I think we only want to parse if `showCustomSlot` is a string though? It could be a TSX Element.